azkaban-common/build.gradle

```diff
@@ -72,6 +72,13 @@ dependencies {
   testRuntime('com.h2database:h2:1.4.193')
 }
 
+task copySqlFiles(type: Copy) {
+  from project(':azkaban-sql').file('src/sql')
```

Comment:
I think we can depend on other projects directly. What is the advantage of this copy?